tests/baselines/reference/parenthesizedContexualTyping1.types

```diff
@@ -130,100 +130,100 @@ var h = fun((((x => x))), ((x => x)), 10);
 
 // Ternaries in parens
 var i = fun((Math.random() < 0.5 ? x => x : x => undefined), 10);
->i : any
->fun((Math.random() < 0.5 ? x => x : x => undefined), 10) : any
+>i : unknown

```

Comment:
This one is funny - `unknown` is better here because it's the constraint of the type parameter - `any` only snuck in here before because it'd get an implicit `any` during the first overload (since the position takes a `T` and not a function in the first overload), which'd then be carried forward into the following overloads. Definitely a safety improvement, but only really noticeable for people with `noImplicitAny` off.